src/styles/header.scss

```diff
@@ -0,0 +1,53 @@
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
```

Comment:
The color value '#0000000d' seems to be using an incorrect format. The correct format for a color with alpha transparency in hexadecimal is '#RRGGBBAA'. Please ensure the alpha value is correctly formatted.